packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -122,13 +110,10 @@ export function PanelChrome({
         </h6>
       )}
 
-      <PanelDescription description={description} className={dragClassCancel} />
-
-      {titleItems !== undefined && (
-        <div className={cx(styles.titleItems, dragClassCancel)} data-testid="title-items-container">
-          {titleItems}
-        </div>
-      )}
+      <div className={cx(styles.titleItems, dragClassCancel)} data-testid="title-items-container">
+        {titleItems}
+        <PanelDescription description={description} className={dragClassCancel} />
```

Comment:
It seems the PanelDescription doesn't need the className since it's wrapped in a div which has the dragClassCancel

Reply:
Edit: Why move the description last?
<img width="966" alt="image" src="https://user-images.githubusercontent.com/13227501/221795083-e2081fd2-e7f2-4948-867c-1e702250ec90.png">

Reply:
Ah! Nice catch, my mistake :)

Reply:
pushed fix!